macos/Sources/Features/QuickTerminal/QuickTerminalWindow.swift

```diff
@@ -1,6 +1,8 @@
 import Cocoa
 
 class QuickTerminalWindow: NSWindow {
+    var focusFollowsMouse: Bool = false
```

Comment:
I think we should add this to `BaseTerminalController` and use that instead of a window property. That would simplify everything else in SurfaceView if we introduce new terminal types.